go/cmd/vtcombo/tablet_map.go

```diff
@@ -161,11 +168,21 @@ func initTabletMap(ts topo.Server, topology string, mysqld mysqlctl.MysqlDaemon,
 		}
 	}
 
-	// Register the tablet dialer
+	// Register the tablet dialer for tablet server
 	tabletconn.RegisterDialer("internal", dialer)
 	*tabletconn.TabletProtocol = "internal"
+
+	// Register the tablet dialer for tablet manager
```

Comment:
nit: You could have written here "tablet manager dialer" instead to distinguish between the two?